client/pages/home.jsx

```diff
@@ -40,27 +42,32 @@ export default function Home(props) {
       .catch(err => console.error(err));
   }, []);
 
-  return (
-    <>
-    <main>
-      <Container maxWidth="lg" >
-        <Typography component="h1" variant="h2" align="center" color="textPrimary" gutterBottom className={classes.heading}>
-          Your Events
-        </Typography>
-        <Grid container justifyContent='center'>
-          <a href='#create-events'>
-            <Button variant="contained" color="primary">
-              CREATE AN EVENT
-            </Button>
-          </a>
-        </Grid>
-          {eventList &&
-            <Grid container spacing={4} className={classes.cardgrid}>
-              {eventList.map(event => <SmallEventCard key={event.eventId} eventId={event.eventId} title={event.title} description={event.description} />)}
-            </Grid>
-          }
-      </Container>
-    </main>
-    </>
-  );
+  if (!user) {
+    window.location.hash = '#sign-up';
```

Comment:
You should also return `null` here after updating the hash. Your render method should always return something. `false` and `null` are used to signify "no elements".